src/styles/blocks/stars.css

```
@@ -0,0 +1,42 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 13px;
+  height: 12px;
+  background-image: url(../../images/star.svg);
+  margin: 2px 3.4px;
+}
+
+.stars__star:first-child {
+  margin-left: 2px;
+}
+
+.stars__star:last-child {
+  margin-right: 2px;
+}
+
+.stars__star:nth-child(3) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../../images/star-active.svg);
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../../images/star-active.svg);
+}
```

```suggestion
.stars--1 .stars__star:nth-child(1) {
  background-image: url(../../images/star-active.svg);
}

.stars--2 .stars__star:nth-child(-n + 2) {
  background-image: url(../../images/star-active.svg);
}

.stars--3 .stars__star:nth-child(-n + 3) {
  background-image: url(../../images/star-active.svg);
}

.stars--4 .stars__star:nth-child(-n + 4) {
  background-image: url(../../images/star-active.svg);
}

.stars--5 .stars__star:nth-child(-n + 5) {
  background-image: url(../../images/star-active.svg);
}
```

combine selectors using comma, example below:

.selector1,
.selector2 {
  your styles here...
}